docs/manifest.json

```diff
@@ -1,214 +1,214 @@
         {
-          "title": "API Reference",
-          "caption": "ECP API reference",
-          "icon": "gear",
-          "path": "apidocs"
+          "title": "ImagePullSecrets",
+          "routes": [
+            {
+              "title": "Managing ImagePullSecrets",
+              "path": "/docs/portal/image-pull-secrets/managing-image-pull-secrets.md"
+            },
+            {
+              "title": "Adding an ImagePullSecret",
+              "path": "/docs/portal/image-pull-secrets/adding-an-image-pull-secret.md"
+            },
+            {
+              "title": "Viewing ImagePullSecret Information",
+              "path": "/docs/portal/image-pull-secrets/viewing-image-pull-secret-information.md"
+            },
+            {
+              "title": "Updating an ImagePullSecret",
+              "path": "/docs/portal/image-pull-secrets/updating-image-pull-secret.md"
+            },
+            {
+              "title": "Deleting an ImagePullSecret",
+              "path": "/docs/portal/image-pull-secrets/deleting-an-image-pull-secret.md"
+            }
+          ]
+        },
+        {
+          "title": "CNAMEs",
+          "routes": [
+            {
+              "title": "Managing CNAMEs",
+              "path": "/docs/portal/cnames/managing-cnames.md"
+            },
+            {
+              "title": "Adding a CNAME",
+              "path": "/docs/portal/cnames/adding-a-cname.md"
+            },
+            {
+              "title": "Viewing CNAME Information",
+              "path": "/docs/portal/cnames/viewing-cname-information.md"
+            },
+            {
+              "title": "Updating a CNAME",
+              "path": "/docs/portal/cnames/updating-a-cname.md"
+            },
+            {
+              "title": "Deleting a CNAME",
+              "path": "/docs/portal/cnames/deleting-a-cname.md"
+            }
+          ]
+        },
+        {              
+          "title": "Registry Management",
+          "routes": [
+            {
+              "title": "Managing Registry Projects",
+              "path": "/docs/portal/image-registry/managing-registry-projects.md"
+            },
+            {
+              "title": "Adding New Projects",
+              "path": "/docs/portal/image-registry/adding-projects.md"
+            },
+            {
+              "title": "Viewing Projects",
+              "path": "/docs/portal/image-registry/viewing-projects.md"
+            },
+            {
+              "title": "Viewing Logs",
+              "path": "/docs/portal/image-registry/viewing-logs.md"
+            },
+            {
+              "title": "Managing Repositories Under a Project",
+              "path": "/docs/portal/image-registry/managing-repositories-under-a-project.md"
+            },
+            {
+              "title": "Managing Tags under a Repository",
+              "path": "/docs/portal/image-registry/managing_tags-under-a-repository.md"
```

Comment:
The path to the MD file is incorrect and causes broken link. Correct fle name shall be "managing-tags-under-a-repository.md"

Reply:
Just make [this commit](https://github.com/mileweb/docs-content-ecp/pull/23/commits/b258d3264f8f699d8c809de1f70821dea6dff64b) to fix it. Setting this conversation to resolved.